CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ to view all changes since the `0.2.1` release.
 
 ### Added
 
+- [`Mesh` overhaul with custom vertex attributes][599]
+    - Any vertex attribute can now be added over `mesh.attributes.insert()`.
+    - See `example/shader/mesh_custom_attribute.rs`
```

Comment:
Perhaps the categories are not named as aptly as they should be, but we should try to only list a change once in the changelog. Since it both changes a current interface (and adds things) could you move this back to `# Changes`?